sklearn/neighbors/_base.py

```diff
@@ -725,10 +729,15 @@ def kneighbors_graph(self, X=None, n_neighbors=None,
 
         Returns
         -------
-        A : sparse graph in CSR format, shape = [n_queries, n_samples_fit]
+        A : {sparse matrix, dataframe} of shape (n_queries, n_samples_fit)
```

Comment:
are you sure that the `dataframe` is supported?

Reply:
No, as I said before, csr_matrix returns Numpy's arrays. I corrected with `{array-like, sparse matrix}`, as in other already present docstrings.